### DISCUSSION TITLE
Deleting files / files management

### DISCUSSION THREAD

**Discussion by RawmanGrey** [nocodb/nocodb#1898] [General]
As of v0.90.7.

Deleting rows containing file references now leaves files in system, - that's not good, at least without some kind of user accessible menu to pick already uploaded files as an alternative to upload new file.

Bare minimum to is to have `/api/v1/db/storage/delete` endpoint. With it, removing files referenced in the deleting row can be handled by build-in (default?) webhook. 

However, it's vague to me how those delete (so as update) calls should be configured, - how should exact file be picked.

Now path structure looks dangerous with possibly localized and non-unique 'project_name' and 'table_name'...
{NC_TOOL_DIR = /usr/app/data}/nc/uploads/{orgs}/{project_name}/{table_name}/{column_name}

Any ideas how this feature has to look like?

**Comment by lukashandrick** [comment]
Are there any updates on this one? It seems that NocoDB still doesn't delete uploaded files when deleting it from the record.
